crates/js-sys/src/lib.rs

```diff
@@ -463,6 +504,134 @@ extern "C" {
     pub fn values(this: &Array) -> Iterator;
 }
 
+// Atomics
+#[wasm_bindge]
+extern "C" {
+    #[derive(Clone, Debug)]
+    #[wasm_bindgen(extends = Object)]
+    pub type Atomics;
+
+    /// The static `Atomics.add()` method adds a given value at a given
+    /// position in the array and returns the old value at that position.
+    /// This atomic operation guarantees that no other write happens
+    /// until the modified value is written back.
+    ///
+    /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/add)
+    #[wasm_bindgen(static_method_of = Atomics, catch)]
+    pub fn add(typed_array: &JsValue, index: u32, value: JsValue) -> Result<JsValue, JsValue>;
```

Comment:
Note that idiomatically in our bindings we try to take `&JsValue` wherever possible instead of `JsValue`, but I wonder if perhaps this should take/return `f64`? I believe you can only pass in numbers here, right?

Reply:
(the point of `JsValue` vs `f64` vs something else applies to most methods here I think)